examples/runtime/main.go

```diff
@@ -0,0 +1,148 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"runtime"
+	"time"
+
+	"github.com/performancecopilot/speed"
+)
+
+const (
+	// refresh interval
+	interval = time.Millisecond
+
+	// number of times metrics are updated
+	iter = 1000
```

Comment:
Maybe we shouldn't fix iterations. Simply using `time.Tick` to create a channel and running it in a separate goroutine should have a similar effect